src/lectures/models/intro_models.ipynb

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
<p>add space between "rectangular" and "cuboids" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/earth-env-data-science/earth-env-data-science-book/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Solved</p>